dtfs-central-api/src/services/tfm/deal-cancellation.service.processPendingCancellation.test.ts

```diff
@@ -83,39 +102,47 @@ describe('DealCancellationService', () => {
-
+    it(`should call PortalDealService.updateStatus with ${DEAL_STATUS.CANCELLED} status`, async () => {
       // Act
-      const dealCancellationResponse = await DealCancellationService.processPendingCancellation(dealId, cancellation, auditDetails);
+      await DealCancellationService.processPendingCancellation(dealId, cancellation, auditDetails);
 
       // Assert
-      expect(dealCancellationResponse).toEqual(DealCancellationService.getTfmDealCancellationResponse(mockRepositoryResponse));
+      expect(updatePortalDealStatusMock).toHaveBeenCalledTimes(1);
+      expect(updatePortalDealStatusMock).toHaveBeenCalledWith({ dealId, dealType, auditDetails, newStatus: DEAL_STATUS.CANCELLED });
     });
 
-    it(`it calls PortalDealService.updateStatus with ${DEAL_STATUS.CANCELLED} status`, async () => {
-      // Arrange
-      const cancellation = aDealCancellation();
-
+    it(`should call PortalFacilityRepo.updateManyByDealId with facilityStage ${DEAL_STATUS.CANCELLED} status for each facility`, async () => {
       // Act
       await DealCancellationService.processPendingCancellation(dealId, cancellation, auditDetails);
 
       // Assert
-      expect(updatePortalDealStatusMock).toHaveBeenCalledTimes(1);
-      expect(updatePortalDealStatusMock).toHaveBeenCalledWith({ dealId, dealType, auditDetails, newStatus: DEAL_STATUS.CANCELLED });
+      expect(updatePortalFacilitiesMock).toHaveBeenCalledTimes(1);
+      expect(updatePortalFacilitiesMock).toHaveBeenCalledWith(dealId, { facilityStage: FACILITY_STAGE.RISK_EXPIRED }, auditDetails);
     });
 
-    it(`it calls PortalFacilityRepo.updateManyByDealId with facilityStage ${DEAL_STATUS.CANCELLED} status for each facility`, async () => {
-      // Arrange
-      const cancellation = aDealCancellation();
-
+    it('should call PortalDealService.addGefDealCancelledActivity with the correct params', async () => {
       // Act
       await DealCancellationService.processPendingCancellation(dealId, cancellation, auditDetails);
 
       // Assert
-      expect(updatePortalFacilitiesMock).toHaveBeenCalledTimes(1);
-      expect(updatePortalFacilitiesMock).toHaveBeenCalledWith(dealId, { facilityStage: FACILITY_STAGE.RISK_EXPIRED }, auditDetails);
+      const expectedAuthor = {
+        firstName: UKEF.ACRONYM,
+      };
+
+      expect(addGefDealCancelledActivityMock).toHaveBeenCalledTimes(1);
+      expect(addGefDealCancelledActivityMock).toHaveBeenCalledWith({
+        deal: mockRepositoryResponse.cancelledDeal,
+        author: expectedAuthor,
+        auditDetails,
+      });
+    });
+
+    it('should return the deal cancellation response object', async () => {
+      // Act
+      const dealCancellationResponse = await DealCancellationService.processPendingCancellation(dealId, cancellation, auditDetails);
+
+      // Assert
+      expect(dealCancellationResponse).toEqual(DealCancellationService.getTfmDealCancellationResponse(mockRepositoryResponse));
```

Comment:
could `DealCancellationService.getTfmDealCancellationResponse(mockRepositoryResponse)` be const expected?